eks/cluster-loader/configs/cluster-autoscaler/config.yaml

```diff
@@ -0,0 +1,131 @@
+# In MB
+{{$NODE_FREE_MEM := DefaultParam .NODE_FREE_MEM 3840}}
+
+# Variables
+{{$NODES_PER_NAMESPACE := DivideInt .Nodes $NAMESPACES}}
+{{$PODS_PER_NAMESPACE := MultiplyInt $PODS_PER_NODE $NODES_PER_NAMESPACE}}
+{{$TOTAL_PODS := MultiplyInt $PODS_PER_NODE (MultiplyInt $NODES_PER_NAMESPACE $NAMESPACES)}}
+
+
+# Pod Resource proportional to node to ensure consistent triggering of scale-up
+{{$POD_CPU_MILLI := DivideFloat $NODE_FREE_CPU (MultiplyFloat $PODS_PER_NODE .9)}}
+{{$POD_MEMORY_MB := DivideFloat $NODE_FREE_MEM (MultiplyFloat $PODS_PER_NODE .9)}}
+
+name: simple-bring-up
+namespace:
+  number: 1
+tuningSets:
+- name: Uniform-QPS
+  qpsLoad:
+    qps: {{$QPS}}
+
+steps:
+- name: start-measurements
+  measurements:
+  - Identifier: ScaleUpTimer
+    Method: Timer
+    Params:
+      action: start
+      label: scaleUpTime
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = {{$POD_GROUP_LABEL}}
+      threshold: 1h
+  - Identifier: SchedulingThroughput
+    Method: SchedulingThroughput
+    Params:
+      action: start
+      labelSelector: group = {{$POD_GROUP_LABEL}}
+
+- name: create-pods
+  phases:
+    - namespaceRange:
+        min: 1
+        max: {{$NAMESPACES}}
+      replicasPerNamespace: 1
+      tuningSet: Uniform-QPS
+      objectBundle:
+        - basename: deployment
+          objectTemplatePath: deployment.yaml
+          templateFillMap:
+            Replicas: {{$PODS_PER_NAMESPACE}}
+            Group: {{$POD_GROUP_LABEL}}
+            CpuRequest: {{$POD_CPU_MILLI}}m
+            MemoryRequest: {{$POD_MEMORY_MB}}M
+
+- name: wait-for-scale-up-complete
+- measurements:
+    - Identifier: WaitingForPods
+      Method: WaitForRunningPods
+      Params:
+        labelSelector: group = {{$POD_GROUP_LABEL}}
+        desiredPodCount: {{$TOTAL_PODS}}
+        timeout: 1h
+
+- name: stop-timing-for-scale-up
+- measurements:
+    - Identifier: ScaleUpTimer
+      Method: Timer
+      Params:
+        action: stop
+        label: scaleUpTime
+
+- name: stop-timing-for-scale-down
+- measurements:
+    - Identifier: ScaleDownTimer
+      Method: Timer
+      Params:
+        action: start
+        label: scaleDownTime
+
+- name: delete-pods
+  phases:
+    - namespaceRange:
+        min: 1
+        max: {{$NAMESPACES}}
+      replicasPerNamespace: 0
+      tuningSet: Uniform-QPS
+      objectBundle:
+        - basename: deployment
+          objectTemplatePath: deployment.yaml
+
+- name: wait-for-scale-down-complete
+- measurements:
+    - Identifier: WaitingForPods
+      Method: WaitForRunningPods
+      Params:
+        labelSelector: group = {{$POD_GROUP_LABEL}}
+        desiredPodCount: 0
+        timeout: 1h
+
+- name: stop-timing-for-scale-down
+- measurements:
+    - Identifier: ScaleDownTimer
+      Method: Timer
+      Params:
+        action: stop
+        label: scaleDownTime
+
+- name: stop-measurements
```

Comment:
Can you combine these measurements into the same "stop-scale-down-measurements" section?

Reply:
will do